src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorEditorFactoryService.cs

```diff
@@ -72,7 +40,12 @@ public override bool TryGetDocumentTracker(ITextBuffer textBuffer, out VisualStu
                 return false;
             }
 
-            EnsureTextBufferInitialized(textBuffer);
+            var textBufferInitialized = EnsureTextBufferInitialized(textBuffer);
```

Comment:
Can you call this `TryInitializeTestBuffer` - it's not really an 'Ensure' if it sometimes fails

Reply:
Can do.